app/mailers/notifications.rb

```diff
@@ -3,28 +3,36 @@ class Notifications < ActionMailer::Base
 
   delegate :name, to: :project, prefix: true
 
+  def started(story, owned_by)
+    @story = story
+    @owned_by = owned_by
+
+    mail :to => story.requested_by.email, :from => owned_by.email,
+      :subject => "[#{story.project.name}] Your story '#{story.title}' has been started."
+  end
+
   def delivered(story, delivered_by)
     @story = story
     @delivered_by = delivered_by
 
     mail :to => story.requested_by.email, :from => delivered_by.email,
-      :subject => "[#{story.project_name}] Your story '#{story.title}' has been delivered for acceptance."
```

Comment:
I thought so, but I couldn't make it pass for some reason ('#project_name doesn't exist')